modules/t/xref_parser.t

```diff
@@ -40,6 +40,11 @@ $database->populate(
   "with force please",
 );
 
+
+# UniProt species taxon codes happen to match species ids of the core database
+my $SPECIES_ID = 6239;
+my $SPECIES_NAME = "Caenorhabditis elegans";
```

Comment:
@mkszuba the test database created here is for homo sapiens. You'd probably want to change the test to use a C. elegans core database if you want it to both use a species id 6239 and remain useful - I'm a fan of the second part more.

Reply:
I can change the ID to 9606 if you prefer. As long as it is a valid value, I do not mind.

Reply:
I remember I had to make them match and the default test db I created was for human. If the test passes it doesn't matter what it is.